Created a section on the docs homepage for the implementation guides. (#1582)
EnterpriseDB/docs: advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx

---

> title: '4. Using CipherTrust Transparent Encryption (CTE)'

Reviewer: No need to surround Postgres in parentheses 
No need for the comma before "and"

description: 'Walking through multiple different instances of CTE in use'
---

CTE protects data either at the file level or at the storage device level. A CTE Agent running on the (Postgres) host manages the files behind a GuardPoint by enforcing the policy associated with it, and communicates data access events to the CipherTrust Manager for logging. A GuardPoint is usually associated with a Linux mount point or a Windows volume, but may also be associated with a directory subtree.

**The following diagram shows the CTE architecture.**

<p align="center">
  <img  src="Images/UsingCTE.png">
</p>

### 4.1 Sample User Scenarios

> This section describes sample user scenarios of deploying CTE solutions on Postgres hosts such as 

Reviewer: ```suggestion
This section describes sample user scenarios of deploying CTE solutions on EDB Postgres Advanced Server and EDB Postgres Extended with BDR hosts.
```

> - **EDB Postgres Advanced Server**

Reviewer: No need for the bulleted list

- **EDB Postgres Extended with BDR**

**EDB Postgres Advanced Server (Single Instance)**

1. Install CTE agent on the Postgres host. 

> 2. Login to the Postgres host and stop the postgres server.

Reviewer: Capitalize Postres whenever referring to Postgres server

11. Restart the Postgres server on the Lead Master node as the user `postgres`.  Make sure you are logged in using ssh (not sudo).

> **The following screenshot shows a GuardPoint created for Lead Master in data center A.**

Reviewer: ```suggestion
The following screenshot shows a GuardPoint created for Lead Master in data center A.
```

<p align="center">
  <img  src="Images/SampleUserScenarios4.png">
</p>

> 12. Repeat steps 2 through 11 for postgres and barman nodes in data center B (DC B).

Reviewer: ```suggestion
12. Repeat steps 2 through 11 for Postgres and Barman nodes in data center B (DC B).
```
